src/ctapipe/io/dl2_tables_preprocessing.py

```diff
@@ -17,13 +17,14 @@
 from tables import NoSuchNodeError
 from traitlets import default
 
+from ctapipe.irf.spectra import SPECTRA, Spectra
```

Comment:
Please keep using relative imports, it will work with two dots:

```
from ..irf.spectra import ...
```

However, please consider to not do this at the top-level but rather only where it is needed to not make https://github.com/cta-observatory/ctapipe/issues/2476 worse again